packaging/prover/miden-prover.service

```diff
@@ -0,0 +1,13 @@
+
+[Unit]
+Description=Miden delegated prover
+Wants=network-online.target
+
+[Install]WantedBy=multi-user.target
+
+[Service]
+Type=execExecStart=/usr/bin/miden-proving-service start-worker --host 0.0.0.0 --port 50052 --prover-type transaction
```

Comment:
Are these just example files or are they supposed to be used exactly as-is?
There are other `ProverType`s supported and I wonder how users would go about changing this to a batch or block proving service.

https://github.com/0xMiden/miden-base/blob/d94df0d845156bf21f5e486e876e3bd4c73ae633/bin/proving-service/src/commands/worker.rs#L10-L20